atlp-rwanda/e-comm-team-techtitans-bn
# #184842288 feature : automated postgres backups

#### What does this PR do?
It creates automated backup system for our database
#### What are the relevant pivotal tracker/Trello stories?
https://www.pivotaltracker.com/story/show/184842288
#### How should this be manually tested?
1. Add environment variables necessary for this task(check: .env.example)
1. npm run dev
2. < after 1 minute the backup will start>
3. You will see have a new backup file named with current date-hour-minute style in folder "automated-backups" in the root directory
4. The backup file will be sent on google drive location https://drive.google.com/drive/u/1/folders/17-AOgQ87BkboPgE6KUvveOp9nfpUfjYN
### Relevant screenshot(s)
![image](https://user-images.githubusercontent.com/71440160/234856093-545bbf29-2db6-46b4-a459-40a6ecf3f644.png)
